CHANGELOG.md

```diff
@@ -7,9 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [12.0.1]
+### Removed
```

Comment:
The "Removed" category is for "removed features", see https://keepachangelog.com/en/1.0.0/
A removed feature is typically a breaking change, as someone might be relying on that feature. But this was changed in a patch release and isn't marked as breaking.

Maybe it's better categorized under "changed"? If we think that this isn't something a user of this package would consider to be a feature.

```suggestion
### Changed
```

Reply:
Thanks Mark updated